config/app.go

```diff
@@ -64,31 +50,32 @@ func init() {
 		// The service providers listed here will be automatically loaded on the
 		// request to your application. Feel free to add your own services to
 		// this array to grant expanded functionality to your applications.
-		"providers": []contracts.ServiceProvider{
+		"providers": []foundation.ServiceProvider{
 			&log.ServiceProvider{},
 			&console.ServiceProvider{},
-			&database.ServiceProvider{},
-			&cache.ServiceProvider{},
-			&http.ServiceProvider{},
-			&route.ServiceProvider{},
-			&schedule.ServiceProvider{},
-			&event.ServiceProvider{},
-			&queue.ServiceProvider{},
-			&grpc.ServiceProvider{},
-			&mail.ServiceProvider{},
-			&auth.ServiceProvider{},
-			&hash.ServiceProvider{},
-			&crypt.ServiceProvider{},
-			&filesystem.ServiceProvider{},
-			&validation.ServiceProvider{},
-			&providers.AppServiceProvider{},
-			&providers.AuthServiceProvider{},
-			&providers.RouteServiceProvider{},
-			&providers.GrpcServiceProvider{},
-			&providers.ConsoleServiceProvider{},
-			&providers.QueueServiceProvider{},
-			&providers.EventServiceProvider{},
-			&providers.ValidationServiceProvider{},
+			//&database.ServiceProvider{},
+			//&cache.ServiceProvider{},
+			//&http.ServiceProvider{},
+			//&route.ServiceProvider{},
+			//&schedule.ServiceProvider{},
+			//&event.ServiceProvider{},
+			//&queue.ServiceProvider{},
+			//&grpc.ServiceProvider{},
+			//&mail.ServiceProvider{},
+			//&auth.ServiceProvider{},
+			//&hash.ServiceProvider{},
+			//&crypt.ServiceProvider{},
+			//&filesystem.ServiceProvider{},
+			//&validation.ServiceProvider{},
+			//&providers.AppServiceProvider{},
+			//&providers.AuthServiceProvider{},
+			//&providers.RouteServiceProvider{},
+			//&providers.GrpcServiceProvider{},
+			//&providers.ConsoleServiceProvider{},
+			//&providers.QueueServiceProvider{},
+			//&providers.EventServiceProvider{},
+			//&providers.ValidationServiceProvider{},
```

Comment:
I just modify the three service providers, so I annotated providers that haven't adapted.